backend/src/services/database.ts

```diff
@@ -12,14 +12,13 @@ class Database {
     * we next look for an environment variable called "postgres_password".
     */
     const passwordArgument: string | undefined = parseArgs(process.argv.slice(2))["password"];
-    const passwordEnvironmentVariable: string | undefined = process.env["postgres_password"];
```

Comment:
I can leave postgres_password. It's worth noting though that node-postgres uses PGPASSWORD by default. If we were to not specify the pool programmatically, it would make a pool with the info through PGPASSSWORD, PGUSER, etc. Also environment variables are capitalized by convention (idk why).

With that said i'll add it back in